src/Illuminate/Queue/Worker.php

```diff
@@ -617,7 +618,7 @@ public function getManager()
     /**
      * Set the queue manager instance.
      *
-     * @param  \Illuminate\Queue\QueueManager  $manager
+     * @param  \Illuminate\Contracts\Queue\Factory  $manager
      * @return void
      */
     public function setManager(QueueManager $manager)
```

Comment:
You changed the docblock, but you forgot to change the actual method typehint.

Reply:
Thanks for pointing.